tests/__fixtures__/Rules/newline_before_scenario/fixture.js

```diff
@@ -0,0 +1,559 @@
+    @tag1
+
+    #comment line
+    @tag2
+    Scenario: a scenario
+
+  Rule: a rule
+
+    #comment line
+    @tag1 @tag2
+    Scenario: a scenario`,
+            [],
+        ],
+        [
+            "without Rule: Scenario",
+            `Feature: a feature file
+
+  Scenario: a scenario`,
+            [],
+        ],
+        [
+            "without Rule: Background - Scenario",
+            `Feature: a feature file
+  Background: a background
+
+  Scenario: a scenario`,
+            [],
+        ],
+        [
+            "without Rule: Scenario Outline",
+            `Feature: a feature file
+
+  Scenario Outline: a scenario outline`,
+            [],
+        ],
+        [
+            "without Rule: Background - Scenario Outline",
+            `Feature: a feature file
+  Background: a background
+
+  Scenario Outline: a scenario`,
+            [],
+        ],
+        [
+            "without Rule: multiple scenarios",
+            `Feature: a feature file
+  Background: a background
+
+  Scenario: a scenario
+
+  Scenario Outline: a scenario outline`,
+            [],
+        ],
+        [
+            "without Rule: description - Scenario",
+            `Feature: a feature file
+  Some feature description
+  is here
+
+  Scenario: a scenario`,
+            [],
+        ],
+        [
+            "without Rule: Scenario tags",
+            `Feature: a feature file
+
+  @tag1 @tag2
+  Scenario: a scenario`,
+            [],
+        ],
+        [
+            "without Rule: Scenario multiline tags",
+            `Feature: a feature file
+
+  @tag1
+  @tag2
+  Scenario Outline: a scenario outline`,
+            [],
+        ],
+        [
+            "without Rule: Scenario comments",
+            `Feature: a feature file
+
+  #comment line
+  Scenario: a scenario`,
+            [],
+        ],
+        [
+            "without Rule: description - Scenario tags and comments",
+            `Feature: a feature file
+  feature description here
+  @tag1
+
+  #comment line
+  @tag2
+  Scenario: a scenario`,
+            [],
+        ],
+        [
+            "without Rule: Background - Scenario tags and comments",
+            `Feature: a feature file
+  feature description here
+
+  Background: a background
+    describe a background
+  @tag1
+
+  #comment line
+  @tag2
+  Scenario: a scenario`,
+            [],
+        ],
+        [
+            "Step datatable",
+            `Feature: a feature file
+  Background: a background
+    Given a step
+      | data1 |
+      | data2 |
+
+  Scenario: a scenario`,
+            [],
+        ],
+        [
+            "Step docstring",
+            `Feature: a feature file
+  Background: a background
+    Given a step
+      """
+      multiline
+      content
+      """
+
+  Scenario: a scenario`,
+            [],
+        ],
+        [
+            "Examples table",
+            `Feature: a feature file
+
+  Scenario: a scenario
+    Given a step
+  Examples:
+      | row1 |
+      | row2 |
```

Comment:
This is a bit strange, it is a `Scenario` but it has `Examples`
I suppose that will pass the `newline_before_scenario` rule, but will fail some rule about "Scenario must not have Example table"